glossary.yml

```
@@ -628,6 +628,11 @@
       Αλγόριθμος είναι ένα σύνολο από βήματα ή κανόνες για τη διεκπεραίωση μιας συγκεκριμένης εργασίας.
       Στην πληροφορική, αλγόριθμος είναι ένα σύνολο από εντολές σε κάποιο πρόγραμμα οι οποίες επιλύουν κάποιο
       υπολογιστικό πρόβλημα.
+  it:
+    term: "algoritmo"
+    def: >
+      Un algoritmo è una sequenza finita di regole ed operazioni da eseguire al fine di realizzare un certo compito.
```

```suggestion
      Un algoritmo è una sequenza finita di operazioni da eseguire al fine di realizzare un certo compito.
```

Non penso serva parlare di regole, poiché un algoritmo non può semplicemente specificare cosa fare senza descrivere come, altrimenti non sarebbe eseguibile.

Ad esempio,
regola: non si possono accettare liste vuote
istruzione: controllare se la lista è vuota e lanciare un'eccezione in caso affermativo